lib/osf-components/addon/components/file-browser/add-new/upload/template.hbs

```diff
@@ -0,0 +1,85 @@
+
+{{#let (unique-id 'upload-files-dropzone') as |id|}}
+    <DropzoneWidget
+        @buildUrl={{this.buildUrl}}
+        @options={{this.dropzoneOptions}}
+        @dropzone={{false}}
+        @addedfile={{action this.addedFile}}
+        @success={{action this.success}}
+        @error={{action this.error}}
+        @enable={{true}}
```

Comment:
I'm guessing this will be tied to a permissions property once I have my work on the providers done?

Reply:
Yes.

Reply:
We could also disable it for registrations now.